auth/api/v1/api_test.go

```diff
@@ -155,9 +156,7 @@ func TestWrapper_GetSignSessionStatus(t *testing.T) {
 
 		err := ctx.wrapper.GetSignSessionStatus(ctx.echoMock, signingSessionID)
 
-		test2.AssertErrProblemTitle(t, problemTitleSignSessionStatus, err)
-		test2.AssertErrProblemStatusCode(t, http.StatusNotFound, err)
-		test2.AssertErrProblemDetail(t, "failed to get session status for 123, reason: session not found", err)
+		test2.AssertIsError(t, err, services.ErrSessionNotFound)
```

Comment:
Not entirely sure about that. The test of `ErrorStatusCodes()` will literally assert the contents of the hardcoded map? Not sure how to test it properly, though. Without making integration tests of course.

Question is also, what would be the impact if it were to break at some point. It'll only return a HTTP 500 instead of the mapped error code (e.g. 400, 403, etc) in case the error occurs. The error response is still returned. So I don't think it's worth it.

Reply:
But now there's no test if an error is actually correctly mapped. I can change a mapping and an API will return a status code which is not according to spec and no test will fail. In short we can break stuff without tests failing.